packages/scripts/README.md

```diff
@@ -287,4 +287,28 @@ Should there be any situation where you want to provide your own webpack config,
 * the command receives a `--config` argument. Example: `wp-scripts build --config my-own-webpack-config.js`.
 * there is a file called `webpack.config.js` or `webpack.config.babel.js` in the top-level directory of your package (at the same level than your `package.json`).
 
+##### Extending the webpack config
+
+To extend the provided webpack config, or replace subsections within the provided webpack config, you can provide your own `webpack.config.js` file, `require` the provided `webpack.config.js` file, and use the [`spread` operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) to import all of or part of the provided configuration.
+
+In the example below, a `webpack.config.js` file is added to the root folder extending the provided webpack config to include [`@svgr/webpack`](https://www.npmjs.com/package/@svgr/webpack) and [`url-loader`](https://github.com/webpack-contrib/url-loader):
+
+```javascript
+const defaultConfig = require("./node_modules/@wordpress/scripts/config/webpack.config");
+
+module.exports = {
+  ...defaultConfig,
+  module: {
+    ...defaultConfig.module,
+    rules: [
+      ...defaultConfig.module.rules,
+      {
+        test: /\.svg$/,
+        use: ["@svgr/webpack", "url-loader"]
+      }
+    ]
+  }
+};
+```
+
```

Comment:
A similar version of this was in the original PR, but we decided against it as to not promote this practice. See also relevant [core-js](https://make.wordpress.org/core/2019/03/13/javascript-chat-summary-march-12th-2019/) discussions. It is already proved convenient for others, though, so we may want to revisit that decission and document this while we figure out a better way to extend it.

I'd like us to add a notice along these lines:

```suggestion
If you follow this approach, please, be aware that future versions of this package may change what webpack and Babel plugins we bundle, default configs, etc. Should those changes be necessary, they will be registered in the [package's CHANGELOG](https://github.com/WordPress/gutenberg/blob/master/packages/scripts/CHANGELOG.md), so make sure to read it before upgrading.
```